portfolio/src/main/webapp/math.js

```diff
@@ -17,6 +18,7 @@ window.onload = function setUpGame() {
 
 function startGame() {
     question.textContent = generateQ();
+    //Remove intro screen and show questions when game starts
```

Comment:
Move this comment to the top of the function. It seems to be explaining what the entirety of the function does, not just the next line of code.

Reply:
Good point, moved and fixed formatting.